bindings/py/setup.py

```diff
@@ -172,19 +177,21 @@ def getPlatformInfo():
   else:
     bitness = "32"
 
+  print "Platform: ",platform,bitness
   return platform, bitness
 
 
 
-def getCompilerInfo():
+def getCompilerInfo(options):
   """
   Identify compiler
   """
 
-  cxxCompiler = ccompiler.get_default_compiler()
-  if "msvc" in cxxCompiler:
-    cxxCompiler = "MSVC"
-  elif "clang" in cxxCompiler:
+  cxxCompiler = getCommandLineOption("compiler", options)
```

Comment:
Probably better to call `getCommandLineOption` in the main function and pass `cxxCompiler` into this function instead of `options`